app/components/article-featured-video.js

```diff
@@ -23,7 +25,37 @@ export default Component.extend(
 		didRender() {
 			this._super(...arguments);
 
-			this.initVideoPlayer();
+			if (!this.get('hasRendered')) {
+				this.initVideoPlayer();
+				this.initOnScrollBehaviour();
+				this.set('hasRendered', true);
+			}
+		},
+
+		/**
+		 * Manages video transformation on user's scroll action
+		 *
+		 * @returns {void}
+		 */
+		initOnScrollBehaviour() {
+			let $video = this.$('.video-container'),
+				videoBottomPosition = $video.offset().top + $video.height();
+
+			this.set('onScrollHandler', () => {
+				run.throttle(this, () => {
+					let currentScroll = this.$(window).scrollTop();
+
+					if (currentScroll >= videoBottomPosition && this.canVideoDrawerShow()) {
+						this.set('isVideoDrawerVisible', true);
+						this.toggleSiteHeadShadow(false);
+					} else if (currentScroll < videoBottomPosition - $video.height()) {
+						this.set('videoDrawerDismissed', false);
+						this.closeVideoDrawer();
+					}
+				}, 200);
+			});
+
+			this.$(window).on('scroll', this.get('onScrollHandler'));
```

Comment:
what do you think of using a namespaced event scroll.featured-video
This way you don't have to pass function to off function

https://api.jquery.com/event.namespace/